packages/ecl-fractal-theme/assets/scss/skins/default.scss

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
`// TODO: use variables from ecl-components-preset-base` might be a good candidate to do in this ticket as well.